app/lib/game-libraries.coffee

```diff
@@ -0,0 +1,9 @@
+# TODO: Webpack: get these from NPM modules where possible
+createjs = require 'lib/createjs-parts'
+window.SPE = require('exports-loader?SPE!imports-loader?THREE=three!vendor/scripts/ShaderParticles')
+require('vendor/scripts/deku')
+require('vendor/scripts/htmlparser2')
+require('vendor/scripts/css')
+# require('vendor/scripts/firepad') # ???
+require('vendor/scripts/jquery-ui-1.11.1.custom')
+require('vendor/scripts/coffeescript')
```

Comment:
I think Firepad is not used any more, needs confirmation. Does not exist in repo or DB code so probably OK to destroy.